### ISSUE TITLE
Static Page: /data/genome_stats/? or /community/genome_info/?

### ISSUE THREAD

**Issue by jd-campbell** [soybase/jekyll-soybase#20]

**Comment by jd-campbell** [comment]
Need to Discuss:  URL location on dev for https://soybase.org/genome_assemblies.php
@StevenCannon-USDA and @maxglycine 

I was thinking either /data/genome_stats/ or /community/genome_info

**Comment by StevenCannon-USDA** [comment]
Addressed in https://github.com/soybase/jekyll-soybase/pull/24

See /resources/genome_info/ and /resources/genome_info/

**Comment by maxglycine** [comment]
That was interesting.  Inserting the yml in the page as apposed to putting it into _data.  But as a policy, I think it would be better to tell workers that all data is in _data.  Also if the list gets really long it might detract from the actual liquid code in the page.

**Comment by StevenCannon-USDA** [comment]
Yeah, I'm of two minds about it. I started by putting all of the data for that page into _data/genome-info-tables/, and then moved the four little "details" tables into the front matter section -- partly just to experiment with that approach. On the one hand, putting all data into _data would be a nice simple rule; on the other hand, putting some small, page-specific data into the front matter would reduce the amount of clutter in _data. At this point, I don't feel strongly one way or the other. I'm with you on this point though: "if the list gets really long it might detract from the actual liquid code in the page." For that reason, I did put the two larger data files into _data/genome-info-tables -- actual as tsv tabular data, which is tidier in this case than yml.
